app/controllers/admin/reports/system-logs/notification-logs.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  columns: [
+    {
+      propertyName : 'user',
```

Comment:
We must use the correct propertyName here. For displaying the user's name we must have `user.first-name` here.